src/DwitTech.AccountService.WebApi/Controllers/UserController.cs

```diff
@@ -1,32 +1,50 @@
-﻿using Microsoft.AspNetCore.Mvc;
-using DwitTech.AccountService.Core.Interfaces;
-
-
-namespace DwitTech.AccountService.WebApi.Controllers
-{
-    public class UserController : BaseController
-    {
-        private readonly IActivationService _activationService;
-
-        public UserController(IActivationService activationService)
-        {
-           
-            _activationService = activationService;
-        }
-
-        
-        [HttpGet("/Activation/{activationCode}")]
-        public async Task<IActionResult> ActivateUser(string activationCode)
-        {
-            try
-            {
-                var activationResult = await _activationService.ActivateUser(activationCode);
-                return Ok(activationResult);
-            }
-            catch (Exception ex)
-            {
-                return BadRequest($"Something went wrong, due to {ex.Message}, please try again");
-            }
-        }    
-    }
-}
+using Microsoft.AspNetCore.Mvc;
+using DwitTech.AccountService.Core.Interfaces;
+
+namespace DwitTech.AccountService.WebApi.Controllers
+{ 
+    public class UserController : BaseController
+    {
+        private readonly IActivationService _activationService;
+        private readonly IUserService _userService;
+
+        public UserController(IActivationService activationService, IUserService userService)
+        {
+
+            _activationService = activationService;
+            _userService = userService;
+        }
+
+
+        [HttpGet("/Activation/{activationCode}")]
+        public async Task<IActionResult> ActivateUser(string activationCode)
+        {
+            try
+            {
+                var activationResult = await _activationService.ActivateUser(activationCode);
+                return Ok(activationResult);
+            }
+            catch (Exception ex)
+            {
+                return BadRequest($"Something went wrong, due to {ex.Message}, please try again");
+            }
+
+        }
+
+        [HttpPost]
+        [Route("login")]
+        public async Task<IActionResult> AuthenticateUserLogin([FromBody] string email, string hashedPassword)
+        {
+            try
+            {
+                var loginResult = await _userService.AuthenticateUserLogin(email, hashedPassword);
```

Comment:
You aren't checking the login result if it is successful?

Reply:
> You aren't checking the login result if it is successful?

Resolved